### PR TITLE
Unpin mkdocs-material-insiders

### PR DESCRIPTION
**Pull request by madkinsz**
This reverts #9499

### CODE REVIEW

**Comment by madkinsz** [comment]
Still issues

```
2:42:19 PM:   File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
2:42:19 PM:     return _bootstrap._gcd_import(name[level:], package, level)
2:42:19 PM:   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
2:42:19 PM:   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
2:42:19 PM:   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
2:42:19 PM:   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
2:42:19 PM:   File "<frozen importlib._bootstrap_external>", line 848, in exec_module
2:42:19 PM:   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
2:42:19 PM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 56, in <module>
2:42:19 PM:     class SocialPlugin(BasePlugin[SocialConfig]):
2:42:19 PM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 641, in SocialPlugin
2:42:19 PM:     def _resolve_font(self, family: str, styles: set[str]):
2:42:19 PM: TypeError: 'type' object is not subscriptable
```

**Review by serinamarie — APPROVED** [review]

**Comment by discdiver** [comment]
Do we want to comment out social cards in mkdocs.insiders.yaml to get it to build a 2.10.8 docs version? Leads to a little weirdness in previews e.g. in Slack I believe.

**Comment by madkinsz** [comment]
@discdiver we can pin to the commit as I did in #9499 if we want to get it released — I'm okay to wait until it's fixed upstream though.

**Review by discdiver — APPROVED** [review]
Specified main branch so matches PrefectHQ/docs

File: netlify.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ base = ""
 command = """
 pip install --upgrade pip \
 && pip install --upgrade --upgrade-strategy eager  -e ".[dev]" \
-&& pip install git+https://oauth:${MKDOCS_MATERIAL_INSIDERS_REPO_RO}@github.com/PrefectHQ/mkdocs-material-insiders.git@dc2fd1973cbe73933f270829520bd6626510de5c \
+&& pip install git+https://oauth:${MKDOCS_MATERIAL_INSIDERS_REPO_RO}@github.com/PrefectHQ/mkdocs-material-insiders.git@main \
```
Comment (madkinsz) [inline netlify.toml:13]:
```suggestion
&& pip install git+https://oauth:${MKDOCS_MATERIAL_INSIDERS_REPO_RO}@github.com/PrefectHQ/mkdocs-material-insiders.git@master \
```

**Review by discdiver — APPROVED** [review]

**Pull request merged**